KronorApi/Schema/InputObjects/PaymentSessionInput.graphql.swift

```diff
@@ -15,6 +15,7 @@ public extension KronorApi {
     public init(
       additionalData: GraphQLNullable<PaymentSessionAdditionalData> = nil,
       amount: Int,
+      country: GraphQLNullable<GraphQLEnum<Country>> = nil,
```

Comment:
We are introducing a Country enum here but that is currently not in the changes & does not exist on the main branch